app/views/episodes/_form_status.html.erb

```diff
@@ -35,15 +35,19 @@
       <%= render "form_feeds", form: form, episode: episode %>
     </div>
 
-    <% if episode.persisted? && episode.publish_to_apple? %>
+    <% if episode.persisted? && episode.publish_to_integration?(:apple) %>
```

Comment:
seems like the status display code for megaphone and apple are identical now, so maybe we could just loop through each of those integrations, instead of duplicating the block of code?

Reply:
👍